### DISCUSSION TITLE
Wezterm got lag while running pytorch (or other GPU tasks in python)

### DISCUSSION THREAD

**Discussion by bavo96** [wezterm/wezterm#4392] [General]
When I used Wezterm to code Python and do some tasks related to GPU, the terminal gradually lagged. Everything was slow in the terminal and it caused Google Chrome to shut down, although the GPU VRAM was not full (only about 7gb/24gb). But when I shut down Wezterm, everything's back to normal (Google Chrome didn't crash anymore). Does anyone encounter this scenario? Please help :(

**Comment by bavo96** [comment]
Do you have any ideas about this @wez?

**Reply by wez** [reply]
Not really. Sounds like a resource contention issue.

**Comment by bavo96** [comment]
Yeah, I think you're right. I'm trying to reproduce the issue because it happens really randomly. I'll gather as much information as I can. I'm also using Neovim with some plugins while this issue is happening. This issue is quite annoying :(

**Comment by bavo96** [comment]
Turns out I'm using [tmux-continum](https://github.com/tmux-plugins/tmux-continuum) to continuously save tmux environment which causes the lagging. When I turn it off, everything goes back to normal. So sorry about this issue :(
